### ISSUE TITLE
use different CRS in stage_ldas_ts?

### ISSUE THREAD

**Issue by aappling-usgs** [USGS-R/mda.streams#279]
@jread-usgs, in light of  https://github.com/USGS-CIDA/stream_metab_usa/pull/121#discussion_r70883034

> there's a column in `get_meta('basic')` called `coord_datum`. The values there are almost always NAD83 (723 of those, 1 of WGS84). does that mean we should be using EPSG:4269 for most sites and EPSG: 4326 just for that one WGS84 site? (i'm getting my info from https://www.nceas.ucsb.edu/~frazier/RSpatialGuides/OverviewCoordinateReferenceSystems.pdf)

should/can we specify the CRS when creating the site stencil in `stage_ldas_ts` at https://github.com/USGS-R/mda.streams/blob/3e9791c0d6ce407d53b989f0cc60614e20ce89ad/R/stage_ldas_ts.R#L55?
```
stencil <- simplegeom(lon_lat_df)
```

**Comment by jread-usgs** [comment]
yes, what I think we should do here is do a spatial transform to get all of them into the datum and proj that `geoknife` uses for the simplegeom class. Good catch.
